ci/deploy_app.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+CONCURRENCY="${CONCURRENCY:=80}"
+CDN_URL="${CDN_URL:=https://cdn.census.gov.uk}"
+NEW_RELIC_APP_NAME="${NEW_RELIC_APP_NAME:=}"
```

Comment:
The ordering of the variables should be consistent between the definition and where they get used in the cloud run command.

Reply:
Consistent now, thanks!